cloudinit/util.py

```diff
@@ -1928,10 +1937,11 @@ def mounts():
                     (dev, mp, fstype, opts, _freq, _passno) = mpline.split()
                 else:
                     m = re.search(mountre, mpline)
-                    dev = m.group(1)
-                    mp = m.group(2)
-                    fstype = m.group(3)
-                    opts = m.group(4)
+                    # safe to type-ignore because of the try-except wrapping
```

Comment:
nit: This might be outside the context of this PR and doesn't need to change here, but I think a nicer refactor would be to remove the try/except entirely and add a None check after the `re.search()` call.

Reply:
I have refactored it, dealing with more conditions that the try/except was able to catch.